easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-X11-20180604.eb

```diff
@@ -0,0 +1,1868 @@
+    }),
+    ('penalized', '0.9-51', {
+        'checksums': ['eaa80dca99981fb9eb576261f30046cfe492d014cc2bf286c447b03a92e299fd'],
+    }),
+    ('clusterRepro', '0.5-1.1', {
+        'checksums': ['b7fcb9db0c762a2e6e22a0a0689affd73504c56f13c13b04272a946630334e6f'],
+    }),
+    ('randomForestSRC', '2.6.1', {
+        'checksums': ['e1607fe2f4a1051a526f1ac6117a1930d12b9959cc41a9dcc77646026af0b872'],
+    }),
+    ('sm', '2.2-5.5', {
+        'checksums': ['43e212a14c364b98b10018b56fe0a619ccffe4bde1294e6c45b3eafe7caf82e7'],
+    }),
+    ('pbivnorm', '0.6.0', {
+        'checksums': ['07c37d507cb8f8d2d9ae51a9a6d44dfbebd8a53e93c242c4378eaddfb1cc5f16'],
+    }),
+    ('lavaan', '0.6-2', {
+        'checksums': ['91d5aea1468394f9e2bda773ab51f8a16a198bfc1d8474c1c48bcaba87f9c0be'],
+    }),
+    ('matrixcalc', '1.0-3', {
+        'checksums': ['17e6caeeecd596b850a6caaa257984398de9ec5d2b41ce83c428f112614b9cb0'],
+    }),
+    ('arm', '1.10-1', {
+        'checksums': ['6f1158c9295e65bd649139224497d3356189b931ff143f9b374daae72548776f'],
+    }),
+    ('mi', '1.0', {
+        'checksums': ['34f44353101e8c3cb6bf59c5f4ff5b2391d884dcbb9d23066a11ee756b9987c0'],
+    }),
+    ('visNetwork', '2.0.4', {
+        'checksums': ['0503408b195a812bcf589f8d34f5c3f9a38e92e78b699c3c7d737f7054d5ed15'],
+    }),
+    ('rgexf', '0.15.3', {
+        'checksums': ['2e8a7978d1fb977318e6310ba65b70a9c8890185c819a7951ac23425c6dc8147'],
+    }),
+    ('influenceR', '0.1.0', {
+        'checksums': ['4fc9324179bd8896875fc0e879a8a96b9ef2a6cf42a296c3b7b4d9098519e98a'],
+    }),
+    ('downloader', '0.4', {
+        'checksums': ['1890e75b028775154023f2135cafb3e3eed0fe908138ab4f7eff1fc1b47dafab'],
+    }),
+    ('DiagrammeR', '1.0.0', {
+        'checksums': ['2b186dae1b19018681b979e9444bf16559c42740d8382676fbaf3b0f8a44337e'],
+    }),
+    ('sem', '3.1-9', {
+        'checksums': ['4a33780202506543da85877cd2813250114420d6ec5e75457bc67477cd332cb9'],
+    }),
+    ('jpeg', '0.1-8', {
+        'checksums': ['d032befeb3a414cefdbf70ba29a6c01541c54387cc0a1a98a4022d86cbe60a16'],
+    }),
+    ('network', '1.13.0.1', {
+        'checksums': ['60b71983b638c7812ccd986ead438ed7bb2bd8080c01c85d8ea70587275e73ad'],
+    }),
+    ('statnet.common', '4.1.4', {
+        'checksums': ['2cea993a52f27d8a5a5e91d3a5dcffcc86523001f6f646cd26ec37c2486dbe91'],
+    }),
+    ('sna', '2.4', {
+        'checksums': ['2292b3190e8d879e494527ae9d9d1174c31cb4183749ecb455aedd8d534048cf'],
+    }),
+    ('glasso', '1.10', {
+        'checksums': ['e6fa74139a2b5f475f134cc0d6b4000ed870beb5865294e9f1d68b4d42bf505b'],
+    }),
+    ('huge', '1.2.7', {
+        'checksums': ['042b771c334cf4d1a92c5c546ca025238919f772a50457594b7e0bd243498d8c'],
+    }),
+    ('d3Network', '0.5.2.1', {
+        'checksums': ['5c798dc0c87c6d574abb7c1f1903346e6b0fec8adfd1df7aef5e4f9e7e3a09be'],
+    }),
+    ('ggm', '2.3', {
+        'checksums': ['832ffe81ff87c6f1a6644e689ebbfb172924b4c4584ac8108d1244d153219ed8']
```

Comment:
@fizwit Please also include the additional extensions we included recently for R 3.4.4 & 3.5.0, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6686/files